contracts/test/ChainLinkOracle.t.sol

```diff
@@ -2,41 +2,51 @@
 pragma solidity 0.6.11;
 pragma experimental ABIEncoderV2;
 
-import { TestUtil } from "../../../../test/TestUtil.sol";
+import { DSTest } from "../../modules/ds-test/src/test.sol";
+import { SecurityAdmin } from "./accounts/SecurityAdmin.sol";
+import { Commoner } from "./accounts/Commoner.sol";
+import { ChainlinkOracle } from "../ChainlinkOracle.sol";
+
+contract ChainlinkOracleTest is DSTest {
+
+    ChainlinkOracle   orcl;
+    SecurityAdmin   sadmin;
+    Commoner           cam;
+
+    address constant wethAggregator = 0x5f4eC3Df9cbd43714FE2740f5E3616155c5b8419;
+    address constant WETH           = 0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2;
```

Comment:
We can mock these

Reply:
I am ok to keep them.

Reply:
If we make a `MockChainlinkAggregatorV3`, then we just need a dummy `latestRoundData` and `seLatestRoundData`, and we can get rid of mainnet `wethAggregator` and `WETH`.